part-i-getting-started/quick-start.md

```diff
@@ -0,0 +1,355 @@
+
+You can download the Axon Framework from our downloads page: [axonframework.org/download](http://www.axonframework.org/download).
+
+This page offers a number of downloads. Typically, you would want to use the latest stable release. However, if you're eager to get started using the latest and greatest features, you could consider using the snapshot releases instead. The downloads page contains a number of assemblies for you to download. Some of them only provide the Axon library itself, while others also provide the libraries that Axon depends on. There is also a "full" zip file, which contains Axon, its dependencies, the sources and the documentation, all in a single download.
+
+If you really want to stay on the bleeding edge of development, you can clone the Git repository: [git://github.com/AxonFramework/AxonFramework.git](git://github.com/AxonFramework/AxonFramework.git), or visit [https://github.com/AxonFramework/AxonFramework](https://github.com/AxonFramework/AxonFramework) to browse the sources online.
+
+## Configure Maven
+
+If you use maven as your build tool, you need to configure the correct dependencies for your project. Add the following code in your dependencies section:
+
+```markup
+<dependency>
+    <groupId>org.axonframework</groupId>
+    <artifactId>axon-core</artifactId>
+    <version>${axon.version}</version>
+</dependency>
+```
+
+Most of the features provided by the Axon Framework are optional and require additional dependencies. We have chosen not to add these dependencies by default, as they would potentially clutter your project with artifacts you don't need.
+
+## Infrastructure requirements
+
+Axon Framework doesn't impose many requirements on the infrastructure. It has been built and tested against Java 8, making that more or less the only requirement.
+
+Since Axon doesn't create any connections or threads by itself, it is safe to run on an Application Server. Axon abstracts all asynchronous behavior by using `Executor`s, meaning that you can easily pass a container managed Thread Pool, for example. If you don't use a full blown Application Server \(e.g. Tomcat, Jetty or a stand-alone app\), you can use the `Executors` class or the Spring Framework to create and configure Thread Pools.
+
+## Let's build our first Axon application
+
+As our domain we'll take a Giftcard management. See the [wikipedia article](https://en.wikipedia.org/wiki/Gift_card) for a basic definition of gift cards. Essentially, there are just two events in the life cycle of a gift card:                                               
+* They get issued: a new gift card gets created with some amount of money stored.
+* They get redeemed: all or part of the monetary value stored on the gift card is used to purchase something.
+
+### Commands and Events
+As mentioned, we need a command to issue the Giftcard. This command will contain the identifier of the Giftcard and the initial amount. Let's build one:
+
+```java
+public class IssueCmd {
+
```

Comment:
For the quick start it's kinda cumbersome, but I personally recommend users to have equals/hashcode/tostring functions in their core api objects, so also the commands and events.
Maybe a short not describing this might be valuable.